### PR TITLE
zebra-puzzle: convert description to introduction and instructions

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/zebra-puzzle/introduction.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Introduction
+
+The Zebra Puzzle is a famous logic puzzle in which there are five houses, each painted a different color.
+The houses have different inhabitants, which have different nationalities, own different pets, drink different beverages and smoke different brands of cigarettes.
+
+To help you solve the puzzle, you're given 15 statements that constrain what solutions are valid.
+However, only by combining the information in _all_ statements will you be able to find the solution to the puzzle.
+
+~~~~exercism/note
+The Zebra Puzzle is a [Constraint satisfaction problem][constraint-satisfaction-problem].
```
Comment (iHiD) [inline exercises/zebra-puzzle/introduction.md:10]:
```suggestion
The Zebra Puzzle is a [Constraint satisfaction problem (CSP)][constraint-satisfaction-problem].
```

File: exercises/zebra-puzzle/introduction.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+In such a problem, you have a set of possible values and a set of constraints that limit which values are valid.
+Another well-known Constraint satisfaction problem are Suduko puzzles.
```
Comment (iHiD) [inline exercises/zebra-puzzle/introduction.md:12]:
```suggestion
Another well-known CSP is Sudoku.
```

File: exercises/zebra-puzzle/instructions.md [hunk 3]
```diff
@@ -18,7 +25,8 @@ Solve the zebra puzzle.
 14. The Japanese smokes Parliaments.
 15. The Norwegian lives next to the blue house.
 
-Each of the five houses is painted a different color, and their inhabitants are of different national extractions, own different pets, drink different beverages and smoke different brands of cigarettes.
+Additionally, each of the five houses is painted a different color, and their inhabitants are of different national extractions, own different pets, drink different beverages and smoke different brands of cigarettes.
 
-Which of the residents drinks water?
-Who owns the zebra?
+~~~~exercism/note
+There are 24 billion (5!^5 = 24,883,200,000) possible solutions, so try ruling out as many solutions as possible.
```
Comment (MatthijsBlom) [inline exercises/zebra-puzzle/instructions.md:31]:
```suggestion
There are 24 billion (5!⁵ = 24,883,200,000) possible solutions, so try ruling out as many solutions as possible.
```

File: exercises/zebra-puzzle/introduction.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Introduction
+
+The Zebra Puzzle is a famous logic puzzle in which there are five houses, each painted a different color.
+The houses have different inhabitants, which have different nationalities, own different pets, drink different beverages and smoke different brands of cigarettes.
+
+To help you solve the puzzle, you're given 15 statements that constrain what solutions are valid.
+However, only by combining the information in _all_ statements will you be able to find the solution to the puzzle.
```
[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply exercises/zebra-puzzle/introduction.md:7]: The idea was that I try to explain what makes the zebra puzzle so fun, in that you have to combine all the statements. But I'm happy for a suggestion on how to phrase that differently.

[the remark this answers is not in the document]
Comment (MatthijsBlom) [reply exercises/zebra-puzzle/introduction.md:7]: Alright 👍

I thought there might be a technical reason that I was overlooking.

[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply exercises/zebra-puzzle/introduction.md:7]: What about:

> To help you solve the puzzle, you're given 15 statements describing the solution.
> However, only by combining the information in _all_ statements will you be able to find the solution to the puzzle.

[the remark this answers is not in the document]
Comment (MatthijsBlom) [reply exercises/zebra-puzzle/introduction.md:7]: I think that is an improvement.

[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply exercises/zebra-puzzle/introduction.md:7]: Updated

**Review by MatthijsBlom — APPROVED** [review]

**Review by BethanyG — APPROVED** [review]

**Review by BNAndras — APPROVED** [review]

**Pull request merged**